front/lib/client/subscription.ts

```diff
@@ -2,7 +2,8 @@
 // so this is just a way to have that value hardcoded in one place.
 // Changing this value only changes the value displayed on the webapp and the website,
 // not on the Stripe dashboard.
-export const PRO_PLAN_29_COST = 29;
+export const PRO_PLAN_COST_MONTHLY = 29;
```

Comment:
Shouldn't we get this data from Strip at some point?

Reply:
Sure we could, this is now only used on the pricing tables (the rest I already refactored). 
Could be a small Olympus task.